frontend/package.json

```diff
@@ -12,17 +13,20 @@
     "@types/react": "^16.9.0",
     "@types/react-dom": "^16.9.0",
     "@types/react-router-dom": "^5.1.5",
+    "@types/stompjs": "^2.3.4",
     "@types/styled-components": "^5.1.2",
     "monaco-editor": "^0.20.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
+    "stompjs": "^2.3.3",
     "styled-components": "^5.1.1",
     "typeface-roboto": "0.0.75",
     "typescript": "~3.7.2"
   },
   "devDependencies": {
+    "@types/sockjs-client": "^1.1.1",
```

Comment:
Could we move this to dependencies to be consistent with the other `@types` dependencies? Or move all of them to dev dependencies if they're not actually necessary for deployment.

Reply:
Yeah no problem, I moved it to the `"dependencies"` list.